packages/cli/src/cli.ts

```diff
@@ -121,9 +128,13 @@ export function main(originalArgs: readonly string[]): void {
       const hexHash = toHex(hash);
       export class NewDummyClass {};
 
-      const pen = await Secp256k1Pen.fromMnemonic(
-        "zebra slush diet army arrest purpose hawk source west glimpse custom record",
-      );
+      const encoded = toHex(toUtf8(toBase64(toAscii("hello world"))));
```

Comment:
Hahaha...funny to see the output.
Also interesting 2 of the `toX` convert bytes to string, and the other two string to bytes